src/app/data-planes/components/PolicyTypeEntryList.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <AccordionList
+    :initially-open="[]"
+    multiple-open
+  >
+    <AccordionItem
+      v-for="(policyEntry, index) in props.policyTypeEntries"
+      :key="index"
+    >
+      <template #accordion-header>
+        <h3>
+          <PolicyTypeTag :policy-type="policyEntry.type">
+            {{ policyEntry.type }} ({{ policyEntry.matchedPolicies.length }})
+          </PolicyTypeTag>
+        </h3>
+      </template>
+
+      <template #accordion-content>
+        <div class="policy-list">
+          <KTable
+            :fetcher="() => ({ data: policyEntry.matchedPolicies, total: policyEntry.matchedPolicies.length })"
+            :headers="tableHeaders"
+            :cell-attrs="getCellAttributes"
+            disable-pagination
+            is-clickable
+          >
+            <template #source="{ rowValue }">
+              <template v-if="rowValue.name !== null">
+                {{ rowValue.name }}
+
+                <TagList
+                  v-if="Array.isArray(rowValue.tags) && rowValue.tags.length > 0"
+                  class="client-tag-list"
+                  :tags="rowValue.tags"
+                />
+              </template>
+
+              <template v-else>
+                —
+              </template>
+            </template>
+
+            <template #destination="{ rowValue }">
+              <template v-if="rowValue !== null">
+                {{ rowValue }}
+              </template>
+
+              <template v-else>
+                —
+              </template>
+            </template>
+
+            <template #name="{ rowValue }">
+              <template v-if="rowValue !== null">
+                {{ rowValue }}
+              </template>
+
+              <template v-else>
+                —
+              </template>
+            </template>
+
+            <template #origin="{ rowValue }">
+              <router-link :to="rowValue.route">{{ rowValue.name }}</router-link>
+            </template>
+
+            <template #config="{ rowValue, rowKey }">
+              <template v-if="rowValue !== null">
+                <details>
+                  <summary>Conf</summary>
+
+                  <CodeBlock
+                    :id="`${index}-${rowKey}-code-block`"
+                    :code="rowValue"
+                    language="yaml"
+                  />
```

Comment:
Can we maybe get rid of the line number of this? I find the expanded conf a little `bulky` also because it has to fit in a column it's quickly wrapped when there's a lot of space .e.g:

<img width="1732" alt="image" src="https://user-images.githubusercontent.com/702355/203533394-ef34b735-53e4-4b6b-8cbe-c1336fab21bb.png">

Wondering if there's something a bit nicer to be done here

Reply:
Yes can't find of a good option here... Let's see if someone else comes up with something :)

Reply:
I did some small adjustments here: the code block no longer shows the copy button here and I’ve also removed the disclosure widget around it.